src/main/webapp/script.js

```diff
@@ -191,10 +221,18 @@ function createNotificationsElement(notification) {
  * Creates an <li> element with question data. 
  * Each element corresponds to a question to be displayed in the DOM.
  */
-function createQuestionElement(question) {
+function createQuestionElement(question, hasRedirect) {
   const questionElement = document.createElement('li');
   questionElement.setAttribute('class', 'list-group-item');
-  questionElement.innerText = question.title;
+
+  if (hasRedirect) {
+    const questionTitle = document.createElement('a');
+    questionTitle.setAttribute('href', '/question.html?id=' + question.id);
```

Comment:
This is probably something to include in a future PR but I'm just curious - is there a char/word limit for these fields? What if someone type 2039480257239487 words into the question title?

Reply:
yeah, it might be a good idea to set a limit on the form elements, i'll write a TODO

Reply:
turns out it is just an attribute for the html element so I did it right away anyways, I set a limit of 100 chars for the question and 2000 for the body, which takes whitespace into account and I figured that maybe a part of the question might involve many lines of code

Reply:
Oh even better, awesome!